routing/endpointregistry.go

```diff
@@ -8,7 +8,11 @@ import (
 	"github.com/zalando/skipper/eskip"
 )
 
-const defaultLastSeenTimeout = 1 * time.Minute
+const (
+	defaultLastSeenTimeout  = 1 * time.Minute
+	defaultStatsResetPeriod = 1 * time.Minute
+	defaultDiscountFactor   = 0.5
```

Comment:
We should reflect in the operation guide how this values affect healthcheck logic.